apacheconfig/lexer.py

```diff
@@ -381,7 +381,7 @@ def make_lexer(**options):
                             'options': options})
 
     if options.get('useapacheinclude', True):
-        lexer_class = type(six.binary_type('ApacheConfigLexer'),
+        lexer_class = type(str('ApacheConfigLexer'),
```

Comment:
`str` is the correct type here, since Python2 expects a non-unicode string, and Python3 expects a unicode string.